auto_py_to_exe/web/js/interface.js

```diff
@@ -2,6 +2,24 @@
 Handle visual events
 */
 
+// Toggle theme (triggered by clicking moon or sun)
+const toggleTheme = () => {
```

Comment:
When you create `setupTheme` in here, this can be renamed to `_toggleTheme` since it should only be called from here.